### PR TITLE
Adopt the newly defined company style

### PR DESCRIPTION
**Pull request by aborsu**

### CODE REVIEW

**Comment by SagaStalin** [comment]
<KGB>4d70ba95f037e4e63aab0c37b09e1f99120c140b
>style(singlequote): replace all inline strings quotes

Shorten status from 53 to 50 charactersMore info [here](https://github.com/sagacify/komitet-gita-bezopasnosti)

File: kgitb/routes.py [hunk 1]
```diff
@@ -7,58 +7,58 @@
...
-        github.update_status(status_url, "error", "Fix the error!")
+        github.update_status(status_url, 'error', 'Fix the error!')
     else:
-        github.update_status(status_url, "error",
-                             "Fix the {} errors!".format(all_errors))
+        github.update_status(status_url, 'error',
+                             'Fix the {} errors!'.format(all_errors))
 
 
 def github_pr():
```
Comment (SagaStalin) [inline kgitb/routes.py]: MINOR Add a docstring to this function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)

File: kgitb/routes.py [hunk 2]
```diff
@@ -7,58 +7,58 @@
 
 
 def check_pr(commits_url, comments_url, status_url):
```
Comment (SagaStalin) [inline kgitb/routes.py]: MINOR Add a docstring to this function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)

File: kgitb/routes.py [hunk 3]
```diff
@@ -7,58 +7,58 @@
...
+        check_pr(body['pull_request']['commits_url'],
+                 body['pull_request']['comments_url'],
+                 body['pull_request']['statuses_url'])
+        return ''
 
-    return "", 204
+    return '', 204
 
 
 def setup(app):
```
Comment (SagaStalin) [inline kgitb/routes.py]: MINOR Add a docstring to this function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)

File: kgitb/rules/__init__.py [hunk 4]
```diff
@@ -31,14 +31,15 @@
...
     return [tup[1] for tup in getmembers(module, isfunction)
-            if tup[0][0] != "_"]
+            if tup[0][0] != '_']
+
 
 _line_rules = _get_rules(line_rules)
 _status_rules = _get_rules(status_rules)
 
 
 def _is_merge(status_line):
```
Comment (SagaStalin) [inline kgitb/rules/__init__.py]: MINOR Add a docstring to this function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)

File: kgitb/rules/__init__.py [hunk 5]
```diff
@@ -31,14 +31,15 @@
...
-    return status_line.startswith("Merge pull request #")
+    return status_line.startswith('Merge pull request #')
 
 
 def split_lines(commit_message):
```
Comment (SagaStalin) [inline kgitb/rules/__init__.py]: MINOR Add a docstring to this function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)

File: kgitb/github.py [hunk 6]
```diff
@@ -50,27 +50,27 @@ def upsert_comment(url, messages):
...
 def update_comment(comments, body):
-    requests.patch(comments["url"],
-                   params={"access_token": GH_TOKEN},
-                   json={"body": body})
+    requests.patch(comments['url'],
+                   params={'access_token': GH_TOKEN},
+                   json={'body': body})
 
 
 def create_comment(url, body):
```
Comment (SagaStalin) [inline kgitb/github.py]: MINOR Add a docstring to this function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)

File: kgitb/github.py [hunk 7]
```diff
@@ -12,34 +12,34 @@
 
 
 def protect(string):
-    return string.replace("[", r"\[")
+    return string.replace('[', r'\[')
 
 
 def quote(string):
```
Comment (SagaStalin) [inline kgitb/github.py]: MINOR Add a docstring to this function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)

File: kgitb/github.py [hunk 8]
```diff
@@ -12,34 +12,34 @@
...
+    comments = requests.get(url, params={'access_token': GH_TOKEN})
     result = []
     for comment in comments.json():
-        if comment["body"].startswith(HIDDEN):
+        if comment['body'].startswith(HIDDEN):
             result.append(comment)
     return result
 
 
 def delete_comment(comment):
```
Comment (SagaStalin) [inline kgitb/github.py]: MINOR Add a docstring to this function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)

File: kgitb/github.py [hunk 9]
```diff
@@ -12,34 +12,34 @@
...
-    return string.replace("[", r"\[")
+    return string.replace('[', r'\[')
 
 
 def quote(string):
-    return ">" + string.replace("\n", "\n>")
+    return '>' + string.replace('\n', '\n>')
 
 
 def get_commits(url):
```
Comment (SagaStalin) [inline kgitb/github.py]: MINOR Add a docstring to this function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)

File: kgitb/github.py [hunk 10]
```diff
@@ -50,27 +50,27 @@ def upsert_comment(url, messages):
...
 
 def create_comment(url, body):
     requests.post(url,
-                  params={"access_token": GH_TOKEN},
-                  json={"body": body})
+                  params={'access_token': GH_TOKEN},
+                  json={'body': body})
 
 
 def update_status(url, state, message):
```
Comment (SagaStalin) [inline kgitb/github.py]: MINOR Add a docstring to this function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)

File: kgitb/github.py [hunk 11]
```diff
@@ -12,34 +12,34 @@
...
             result.append(comment)
     return result
 
 
 def delete_comment(comment):
-    requests.delete(comment["url"], params={"access_token": GH_TOKEN})
+    requests.delete(comment['url'], params={'access_token': GH_TOKEN})
 
 
 def upsert_comment(url, messages):
```
Comment (SagaStalin) [inline kgitb/github.py]: MINOR Add a docstring to this function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)

File: kgitb/github.py [hunk 12]
```diff
@@ -50,27 +50,27 @@ def upsert_comment(url, messages):
...
             create_comment(url, message)
         if len(comments) == 1:
-            if comments[0]["body"] != message:
+            if comments[0]['body'] != message:
                 update_comment(comments[0], message)
         else:
             create_comment(url, message)
 
 
 def update_comment(comments, body):
```
Comment (SagaStalin) [inline kgitb/github.py]: MINOR Add a docstring to this function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)

File: kgitb/github.py [hunk 13]
```diff
@@ -12,34 +12,34 @@
 
 
 def protect(string):
```
Comment (SagaStalin) [inline kgitb/github.py]: MINOR Add a docstring to this function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)

**Comment by SagaStalin** [comment]
SonarQube analysis reported 20 issues
* MINOR 20 minor

Watch the comments in this conversation to review them.

#### 6 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [__main__.py](https://github.com/Sagacify/komitet-gita-bezopasnosti/blob/8ed505264f596522d6b01bf56c3bada597913239/kgitb/__main__.py): Add a docstring to this module. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)
1. MINOR [__main__.py#L9](https://github.com/Sagacify/komitet-gita-bezopasnosti/blob/8ed505264f596522d6b01bf56c3bada597913239/kgitb/__main__.py#L9): Add a docstring to this function. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)
1. MINOR [config.py](https://github.com/Sagacify/komitet-gita-bezopasnosti/blob/8ed505264f596522d6b01bf56c3bada597913239/kgitb/config.py): Add a docstring to this module. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)
1. MINOR [routes.py](https://github.com/Sagacify/komitet-gita-bezopasnosti/blob/8ed505264f596522d6b01bf56c3bada597913239/kgitb/routes.py): Add a docstring to this module. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)
1. MINOR [line_rules.py](https://github.com/Sagacify/komitet-gita-bezopasnosti/blob/8ed505264f596522d6b01bf56c3bada597913239/kgitb/rules/line_rules.py): Add a docstring to this module. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)
1. MINOR [status_rules.py](https://github.com/Sagacify/komitet-gita-bezopasnosti/blob/8ed505264f596522d6b01bf56c3bada597913239/kgitb/rules/status_rules.py): Add a docstring to this module. [rule](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)

**Review by fbeuvens — APPROVED** [review]

**Pull request merged**